js/CRM_Custom_Form_Field.js

```diff
@@ -22,16 +34,33 @@ CRM.$(function($) {
       $('#autoinc-min-value').show();
       $('#hideDefault, #hideDesc, .crm-custom-field-form-block-is_required, .crm-custom-field-form-block-is_view').hide();
 
+      // If civicrm v5.32.2
+      if ($('#html_type').length) {
+        $(htmlType).attr('disabled', 'disabled');
+      }
+
       // Add Autoincrement text next to Integer if autoinc field is checked
       var $dataTypeField = $('.crm-custom-field-form-block-data_type .crm-frozen-field');
       $dataTypeField.html($dataTypeField.html().replace('Integer', ts('Integer (Autoincrement)')));
     }
 
     // Show autoincfield if data type matches
-    if($('#data_type_0').val() == 1 && $('#data_type_1').val() === 'Text') {
+    if($(dataType).val() == 1 && $(htmlType).val() === 'Text') {
```

Comment:
@lucky091588 It seems odd to use '==' for the first comparison, and then '===' for the second. Could we use '===' in both comparisons?

Reply:
Done

Reply:
Great, thanks.